angularSTEP/src/app/upload-recipe/upload-recipe.component.ts

```diff
@@ -151,77 +179,158 @@ export class UploadRecipeComponent{
   }
 
   removeNumsAndSymbolsAtStart(originalString: string): string {
-    var startIndex: number = -1;
+    const startIndex = -1;
 
     for (let i = 0; i < originalString.length; i++) {
-
       // charCodeAt(i) of 65 is A and of 90 is Z
       // charCodeA(i) of 97 is a and of 122 is z
-      var currentValue: number = originalString.charCodeAt(i);
-      if ((65 <= currentValue && currentValue <= 90) || (97 <= currentValue && currentValue <= 122)) {
+      const currentValue: number = originalString.charCodeAt(i);
+      if (
+        (65 <= currentValue && currentValue <= 90) ||
+        (97 <= currentValue && currentValue <= 122)
+      ) {
         return originalString.substring(i);
       }
     }
-    if (startIndex == -1) {
+    if (startIndex === -1) {
       return '';
     }
     return '';
```

Comment:
What's the purpose of `startIndex` in the first place if it is a constant assigned -1. Then later the result of this `if` test doesn't matter because it could be deleted and still `return '';` in all cases anyways

Reply:
Hm I'm not actually sure. That is not really my code but your logic makes sense so I will delete it.

Reply:
When I was originally creating this function it went through a few different forms as I went through different goals of what the function should accomplish and return, so at one point `startIndex` was necessary, but I guess I forgot to remove it when it was no longer needed and then no one noticed in any previous PRs.